Jenkinsfile

```diff
@@ -0,0 +1,20 @@
+node("docker && awsaccess") {
+  cleanWs()
+  checkout scm
+
+  stage("Build Image") {
+    docker.build("ca_testing")
+  }
+
+  stage("Lint and Unit Test") {
+    withDockerRegistry(registry: [credentialsId: "docker_hub"]) {
```

Comment:
sorry mate, this should be around `docker.build()`. This is when stuff is pulled from docker hub